src/components/plots/barPlot.ts

```diff
@@ -2,6 +2,12 @@
 
 module Plottable {
 export module Plots {
+  export module Bars {
+    export class Mode {
+      public static VERTICAL = "vertical";
+      public static HORIZONTAL = "horizontal";
```

Comment:
Plottable.Plots.Bars.Mode.VERTICAL is quite a mouthful (also didn't we say we only want to go to like a depth of two modules or something like that?) Why don't you just put these constants under Plottable.Plots since most plots can be described as either HORIZONTAL or VERTICAL?